ch04/src/Solution425.java

```diff
@@ -0,0 +1,15 @@
+public class Solution425 {
+    public static void main(String[] args) {
+        final int plateNumbers = (int)(Math.random() * 10000);
+        char[] plateLettersInChar = new char[3];
+
+        for (int i = 0; i < 3; i++) {
+            final char letter = (char)((int)(Math.random() * ((int)'Z' + 1 - (int)'A') + (int)'A'));
+            plateLettersInChar[i] = letter;
+        }
+
+        final String plateLettersInString = String.valueOf(plateLettersInChar);
+
+        System.out.println(plateLettersInString + plateNumbers);

```

Comment:
Usually, vehicle plate numbers have a fixed length. So we write `ABC-0012` instead of `ABC-12`. Can you please modify the program to behave accordingly?